docs/bare-metal/README.md

```diff
@@ -0,0 +1,298 @@
+# Bare-Metal Mantl
+
+By bare-metal we mean a set of physical computers that
+have centos 7 installed on them.
+
+If you are using open-stack, Vmware, or a cloud provider, Mantl.io has terraform scripts for
+you. From a Mantl.io perspective, this doc is about setting up the inventory file
+by hand and preparing the machines to a state similar to what terraform would have
+done.
+
+These scripts were created for set of 8 Intel NUCs. Each had 16GB of RAM
+2 hyper threaded cores (I5s and I3s).  3 of them had 500 GB SSDs and 3
+have 2 TB HDs.  Your hardware can vary.  Theses systems are 4 times larger
+that the AWS systems defined in `terraform/aws.sample.tf` (m3.medium
+instances have 1 core and 4GB RAM).  In other words, your machines could
+be even smaller than our test system.   In fact, maybe you
+have a couple of physical boxes, and then created a few virtual machine vms for the controls
+on a couple of laptops. They have also been used on much larger machines.
+
+This document explains how to prepare your machine with the base OS, network
+hard drive concerns, creating your inventory and getting ansible ready.
+
+## Setting Up Centos 7
+
+### Thumb Drive Install
+
+There are much more professional ways of creating your instances.  But if you are looking for a solution for a
+couple machines at home, perhaps you actually need some tips on how to do it.  The least technical way to do this is
+with a thumb drive.
+
+Create a boot Centos 7 http://www.myiphoneadventure.com/os-x/create-a-bootable-centos-usb-drive-with-a-mac-os-x
+This can be a bit confusing and I used this as well. http://www.ubuntu.com/download/desktop/create-a-usb-stick-on-mac-os
+Mantl.io should start with the current Centos 7 minimum.
+
+During installation you will do the defaults except:
+
+ * manually configure your partitions. ON the Manual partioning page.
+   * Remove existing partitions
+   * Press the button to automatically partition. This will give you a default set.
+   * It will put 50 toward root and the rest in home, change this:
+     * These are services machines and shouldn't really have much if in home.
```

Comment:
Maybe "These machines won't have much data in /home" or "These machines won't store many files in /home"